### PR TITLE
Replaced cStringIO.StringIO by io.BytesIO

### PR DESCRIPTION
**Pull request by claudep**
The cStringIO module is no more in Python 3, and the io module
has been introduced in Python 2.6.

### CODE REVIEW

**Comment by claudep** [comment]
The docs build error is:
```/home/travis/build/translate/translate/translate/misc/wStringIO.py:docstring of translate.misc.wStringIO.CatchStringOutput.seek:5: WARNING: Definition list ends without a blank line; unexpected unindent.```
I'm afraid the issue might be in a Python lib docstring (there are no `translate.misc.wStringIO.CatchStringOutput.seek` method). Any idea?

File: translate/storage/pypo.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 import copy
 import re
 import textwrap
-from cStringIO import StringIO
+from six import BytesIO
```
Comment (unho) [inline translate/storage/pypo.py]: Why not `from io import BytesIO`?

File: translate/convert/prop2mozfunny.py [hunk 2]
```diff
@@ -37,6 +37,8 @@ def prop2inc(pf):
             else:
                 for blank in pendingblanks:
                     yield blank
+                if isinstance(comment, unicode):
+                    comment = comment.encode("UTF-8")
```
Comment (unho) [inline translate/convert/prop2mozfunny.py]: This seems an unrelated change. What is its purpose?

File: translate/storage/pypo.py (hunk 1 above)
Comment (claudep) [inline translate/storage/pypo.py]: Good catch, we don't need six at this stage.

File: translate/convert/prop2mozfunny.py (hunk 2 above)
Comment (claudep) [inline translate/convert/prop2mozfunny.py]: `prop2inc` produces mixed bytestring/unicode lines, which is an issue when using writelines to a bytes file object.

**Comment by unho** [comment]
> I'm afraid the issue might be in a Python lib docstring (there are no translate.misc.wStringIO.CatchStringOutput.seek method). Any idea?

I guess that might be the cause. New commits have been merged to master, so please rebase and lets see if we are lucky and the error goes away.

**Comment by julen** [comment]
> I'm afraid the issue might be in a Python lib docstring (there are no `translate.misc.wStringIO.CatchStringOutput.seek` method). Any idea?

In *docs/api/misc.rst* under the `wStringIO` section try to remove `:inherited-members:`. Such option can cause trouble when docstrings come from 3rd party sources.

**Comment by claudep** [comment]
Yeah, that's what I did to stop Travis complaining!

**Comment by unho** [comment]
@claudep Please rebase this PR so we can cleanly merge it.

**Comment by claudep** [comment]
Done.

**Comment by unho** [comment]
Merged. Thanks!
